index.js

```diff
@@ -47,7 +52,16 @@ datasets.forEach(dataset => {
   deleteEndpointDatasets.forEach(data => {
     app.delete(`${pathPrefix}/${data}`, (request, response) => {
       let newData = request.body;
-      app.locals.bookings = app.locals.bookings.filter(booking => booking.id !== newData.id);
+      app.locals[data] = app.locals[data].filter(el => el.id !== newData.id);
+      return response.sendStatus(200);
```

Comment:
DELETEs work for me too! This looks good so far. The only thing that's tricky is that you get 200 status even if the trip no longer exists. Is that what you intended? I could see it getting a little confusing for students to test